Library/Formula/git-extras.rb

```diff
@@ -20,7 +19,6 @@ class GitExtras < Formula
   end
```

Comment:
Already working on getting it upstream: https://github.com/tj/git-extras/pull/491. Added a link to that, too.